### ISSUE TITLE
Error while raytracing tile

### ISSUE THREAD

**Issue by danielrlutz** [webbukkit/dynmap#3388]
**Issue Description:** The server repeatedly (though with different block in different locations) outputs this message to the server console/log:
```
[Sat, 69. Jun 4200:13:37 UTC ERROR] [dynmap] Error while raytracing tile: perspective=iso_SE_60_lowres, coord=100,65,99, blockid=minecraft:chest[facing=east,type=left,waterlogged=false], lighting=15:9, biome=PLAINS
```

* **Dynmap Version:** 3.1-457
* **Server Version:** 1.16.5-R0.1-SNAPSHOT
* **Pastebin of Configuration.txt:** https://pastebin.com/Xew4NLFh
* **Server Host (if applicable):** selfhosted
* ~~**Pastebin of crashlogs or other relevant logs:** no relevant crashlogs available~~
* ~~**Other Relevant Data/Screenshots:** no additional screenshots or data~~
* ~~**Steps to Replicate:** No obvious reproducing steps~~

 - [x] *I have looked at all other issues and this is not a duplicate* (there is one issue)
 - [x] *I have been able to replicate this*

**Comment by IAmEchino** [comment]
Having the same issue in the new 1.18 experimental build.

Examples include:

`[16:36:42 ERROR]: Error while raytracing tile: perspective=iso_S_90_medres, coord=-629,-2,352, blockid=minecraft:deepslate[axis=y], lighting=0:0, biome=BIOMEBASE@6A87AD63
java.lang.ArrayIndexOutOfBoundsException: null`

and 

`[18:43:05 ERROR]: Error while raytracing tile: perspective=iso_S_90_medres, coord=-48,-7,374, blockid=minecraft:deepslate[axis=y], lighting=0:0, biome=BIOMEBASE@279592DA
java.lang.ArrayIndexOutOfBoundsException: null`

**Comment by mikeprimm** [comment]
Not seen in recent builds - reopen if reproduceable with 3.3 GA

**Comment by IAmEchino** [comment]
> Not seen in recent builds - reopen if reproduceable with 3.3 GA

This issue was resolved in v3.3-beta-3:

![image](https://user-images.githubusercontent.com/64720141/152667507-1c741732-5bae-427c-95aa-c9b1fc33c744.png)

_(Sorry if this reopens the issue, I don't know how to use Github tbh, just doing this so that you can mark the issue as resolved.)_

**Comment by mikeprimm** [comment]
All good - hard for us to keep track of too :).   We hold off on closing just to say 'too old', but it's more just a chance to reopen communications on what MIGHT be a stale issue - if you see something in the current release that looks like this, please feel free to reopen :)
